_docs/_latest/configure/notifier/index.md

```diff
@@ -88,6 +88,22 @@ accordingly.
       data_format: csv
   ```
 
+#### YAML below shows the email connector config for Mailjet.
+
+To configure other email connector, `name` and `auth` fields should be modified
+accordingly.
+
+  ```yaml
+  notifier:
+    email_connector:
+      name: sendgrid
```

Comment:
Should this be something other than `sendgrid`?

Reply:
yes of course for mailjet it should be specified